pytest_fixtures/api_fixtures.py

```diff
@@ -74,11 +74,11 @@ def module_manifest_org():
 
 
 @pytest.fixture(scope='module')
-def module_gt_manifest_org():
+def module_gt_manifest_org(default_sat):
```

Comment:
@mshriver how do we deal with the situation where a test is using `destructive_sat` and need to use `module_gt_manifest_org` fixture?

Reply:
That's outside the scope of this PR, but a great question.

`destructive_sat` really shouldn't be a separate fixture, because of this. The mark should modify the behavior of the fixture, to supply a freshly deployed satellite at the given scope. 

This is due to refactoring these destructive tests over time, and while many helper functions are still in use.